### ISSUE TITLE
Medbot Head Surgeon is VERY loud

### ISSUE THREAD

**Issue by goonstation-issuebot** [goonstation/goonstation#19590]
### Labels

[BUG][MINOR]

### Description

The Head Surgeon medbot voice is extremely loud compared to the other sounds

### Steps to reproduce

1.Medbot speaks
2.Loud

### Expected Behavior

Apparently some people share this issue, and others dont, judging by a discord discussion about this

### Additional Information & Screenshots

Reported by: bilo216
Client version: 515.1635
On server: Goonstation 3 Roleplay: Morty
Active test merges: #19460 and #19331
Round log date: 2024-06-19-03-37
Reported on: 2024-06-19 03:46:08
Round ID: 181771
Map: KONDARU

**Comment by drgg** [comment]
also experiencing this as well as heard a good number of others suffering from it too, only current workaround known is to just.. disable vox and dectalk under audio options

<details>
<summary>my specs</summary>
<br>
Edition	Windows 11 Education
Version	23H2
OS build	22631.3737
Experience	Windows Feature Experience Pack 1000.22700.1009.0
</details>

if anyone picks this up and needs any further info i'll happily provide it

**Comment by BatElite** [comment]
From the looks of it, head surgeon's dectalk uses ehjax.send() which looks to just directly dump the audio to clients. There's no volume control like with the normal playsound()

**Comment by OmegaShoots** [comment]
Yes, I've had my client volume at an acceptable level and every time the medbot speaks it literally buzzes my headphones and hurts my ears when everything else in the game is at enjoyable levels
I've now had to turn down the volume in the system mixer even lower just to account for this one sound and everything is super quiet in turn

**Comment by OmegaShoots** [comment]
After testing change volume with the medbot a few times, it seems tied to the master volume technically, you can change the volume while it's speaking and it'll respect it that one time, however if you leave the area and come back(?) or it just speaks again, it goes back to ignoring your volume setting and blasts audio at max.
